modules/custom/az_paragraphs/az_paragraphs.info.yml

```diff
@@ -11,3 +11,4 @@ dependencies:
   - drupal:text
   - drupal:telephone
   - entity_embed:entity_embed
+  - linkit:linkit
```

Comment:
Nitpick request:
Can we add `az_core` as a module dependency here and move the linkit profile into `modules/custom/az_core/config/optional`?

Reply:
`az_core` is already a dependency in this file

Reply:
Looks like it might not like being in `az_core/config/optional`. Moving back to `config/optional`

Reply:
Do we want this here or in the az_media.info.yml, My preference is az_media, since that is where the az_standard formatter lives.

Reply:
It failed when I had it in `az_media`. Same dependency issue

Reply:
What was the dependency issue?

Reply:
Just "unmet dependencies", even after including `linkit` and `az_core` and `az_paragraphs` as dependencies for the `az_media` module.

Reply:
Which unmet dependencies?